### DISCUSSION TITLE
Can not install on Ububtu/Kubuntu 24.10 Oracular Oriele

### DISCUSSION THREAD

**Discussion by jenders97** [helix-editor/helix#12359] [General] [answered]
### Summary

I just installed Kubuntu on a new system today and went to install Helix. When I added the ppa, I recieved the error below.
```E: The repository 'https://ppa.launchpadcontent.net/maveonair/helix-editor/ubuntu oracular Release' does not have a Release file.```
I believe this is because there isn't a folder for it in "http://ppa.launchpadcontent.net/maveonair/helix-editor/ubuntu/dists/", but I don't know anything about this stuff. I can install from the source, but it would be very nice to not have to.

### Reproduction Steps

Attempt to install helix on ubuntu or kubuntu 24.10 Oracular Oriele

### Platform

Linux

### Terminal Emulator

Konsole Version 24.08.1

### Installation Method

Apt

### Helix Version

N/A

**Comment by pascalkuthe** [comment, marked as answer]
Like all package manager packages the ubuntu PPA si not maintained by us. I don't know who maintains the ubuntu PPA but you would need to contact them.

**Reply by jenders97** [reply]
Ah makes sense. I thought it was official since it had its own PPA. I'll see if I can get in touch with the maintainer. Thanks for the help.
